code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -649,7 +649,7 @@
 /datum/reagent/toxin/heparin/on_mob_life(mob/living/M)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
-		H.blood_max += 2
+		H.bleed_rate = min(H.bleed_rate + 2, 8)
```

Comment:
this is technically a buff. the old code was nonsensical so there's no way for me to tell what was bug and what was intended. the bleeding var was increased here, but handle_blood() was reseting it every time, meaning the heparin code here did nothing. There was a special snowflake code for heparin in handle_blood() but the effect didn't match what's here. So there you go I guess.